qa/src/test/java/io/camunda/migrator/qa/SkipAndRetryProcessInstancesTest.java

```diff
@@ -126,18 +133,61 @@ public void shouldLogWarningWhenProcessInstanceHasBeenCompleted(CapturedOutput o
     // given skipped process instance
     deployProcessInC7AndC8("multiInstanceProcess.bpmn");
     var process = runtimeService.startProcessInstanceByKey("multiInstanceProcess");
-    runtimeMigrator.migrate();
+    runtimeMigrator.start();
     ensureTrue("Unexpected state: one process instance should be skipped", idKeyMapper.findSkippedProcessInstanceIds().size() == 1);
 
     runtimeService.deleteProcessInstance(process.getId(), "State cannot be fixed!");
 
     // when running retrying runtime migration
-    runtimeMigrator.setRetryMode(true);
-    runtimeMigrator.migrate();
+    runtimeMigrator.setMode(RETRY_SKIPPED);
+    runtimeMigrator.start();
 
     // then
     assertThat(output.getOut())
         .containsPattern("WARN(.*)Process instance with legacyId [a-f0-9-]+ doesn't exist anymore. Has it been completed or cancelled in the meantime\\?");
   }
 
+  @Test
+  public void shouldListSkippedProcessInstances(CapturedOutput output) {
+    // given skipped process instance
+    deployProcessInC7AndC8("multiInstanceProcess.bpmn");
+    List<String> processInstancesIds = new ArrayList<>();
+    for (int i = 0; i < 10; i++) {
+      processInstancesIds.add(runtimeService.startProcessInstanceByKey("multiInstanceProcess").getProcessInstanceId());
+    }
+    runtimeMigrator.start();
+    ensureTrue("Unexpected state: one process instance should be skipped", idKeyMapper.findSkippedProcessInstanceIds().size() == 10);
```

Comment:
All 10 are skipped right? Also could we use the count method here?
```suggestion
    ensureTrue("Unexpected state: 10 process instances should be skipped", idKeyMapper. findSkippedProcessInstanceIdsCount() == 10);
```

Reply:
Indeed! FIxed